mmhuman3d/core/conventions/keypoints_mapping/spin.py

```diff
@@ -0,0 +1,26 @@
+SPIN_KEYPOINTS = [
```

Comment:
SPIN uses smpl_49, which is defined in smpl.py

Reply:
SPIN keypoints used in ExPose training is different from SMPL-24. It is now renamed as 'SPIN_SMPLX_KEYPOINTS'.